cmd/daemon/main.go

```diff
@@ -35,7 +35,7 @@ func main() {
 	switch runtimeType {
 	case runtimeTypeContainerd:
 		logger.Info("creating new containerd freezeThawer")
-		freezeThaw, err = containerd.New()
+		freezeThaw, err = containerd.New(&containerd.ContainerdCri{})
```

Comment:
I suspect with this refactor the outer `containerd` might be able to become `freezer` (or something) and we can have the same thing work for both containerd and crio (just with a different Cri interface injected), which would be nice (I don't think we need to do that here, just thinking out loud)

Reply:
A good time to do that may be when I merge these changes into the Docker PR...